packages/host/src/Host.tsx

```diff
@@ -17,6 +17,7 @@ import {
   RandomLoader,
   ConfirmationModalProvider,
   AuthProvider,
+  AlertModalProvider,
```

Comment:
I think soon we need to start grouping some of this? Though.. then we would need to start naming the groups. Uh oh..

Reply:
😱